redbot/core/events.py

```diff
@@ -156,6 +156,11 @@ async def on_ready():
                             python=sys.executable, package_extras=package_extras
                         )
                     )
+                    extra_update += _(
+                        "\nOnce you've started up your bot again, if you've installed some cogs "
+                        "we then highly recommend you update them with this command in Discord: "
+                        "`[p]cog update`"
```

Comment:
In addition to this suggestion, you also need to replace `[p]` with bot's prefix.
To do that, you can call `send_to_owners_with_prefix_replaced()` instead of `bot.send_to_owners()` call in the last line of this function. The `send_to_owners_with_prefix_replaced()` can be imported from `redbot.core.utils._internal_utils` module.
```suggestion
                        "\nOnce you've started up your bot again, if you have any 3rd-party cogs"
                        " installed we then highly recommend you update them with this command"
                        " in Discord: `[p]cog update`"
```